### PR TITLE
Add go link for Impeller Dart doc

### PR DESCRIPTION
**Pull request by bdero**
_Description of what this PR is changing or adding, and why:_

Adds shortlink for a new doc.

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Review by parlough — CHANGES_REQUESTED** [review]
Thanks for adding the redirect.

The doc seems to be private. Can you share by link?

**Comment by bdero** [comment]
This doc is already public: 
image

What happens when you try to access it? https://docs.google.com/document/d/1Sh1BAC5c_kkuMVreo7ymBzPoMzb7lamZRPsI7GBXv5M/edit?usp=sharing

**Comment by parlough** [comment]
I get:

You need access

**Comment by parlough** [comment]
Hmm, I'm able to access it on another Google account though. This time with a [resource key](https://developers.google.com/drive/api/guides/resource-keys) which works on the other accounts - https://docs.google.com/document/d/1Sh1BAC5c_kkuMVreo7ymBzPoMzb7lamZRPsI7GBXv5M/edit?resourcekey=0-5w8u2V-LS41tCHeoE8bDTQ

Perhaps adjust the link to one with the resource key?

File: firebase.json [hunk 1]
```diff
@@ -325,6 +325,7 @@
       { "source": "/go/icu-message-parser", "destination": "https://docs.google.com/document/d/1wj5rBbuz1bXM1lvt0s0d35F5JRtIPmYUs1UDk1G3HLc/edit?usp=sharing", "type": 301 },
       { "source": "/go/idle-gc-metrics", "destination": "https://docs.google.com/document/d/1QjLTDr2cTsmS3DGIWdw9WSgqBMKJkFXD7Q8acziGIIA/edit?usp=sharing", "type": 301 },
       { "source": "/go/image-decoding-registry", "destination": "https://docs.google.com/document/d/167qWrlaSAmJm5muqQ-iiJKZy6Q7ZlugIeXmraXava5Y/edit?resourcekey=0-mVrr_zBiSI2Qmd6vqae7Bw", "type": 301 },
+      { "source": "/go/impeller-dart", "destination": "https://docs.google.com/document/d/1Sh1BAC5c_kkuMVreo7ymBzPoMzb7lamZRPsI7GBXv5M/edit?usp=sharing", "type": 301 },
```
Comment (parlough) [inline firebase.json:328]:
```suggestion
      { "source": "/go/impeller-dart", "destination": "https://docs.google.com/document/d/1Sh1BAC5c_kkuMVreo7ymBzPoMzb7lamZRPsI7GBXv5M/edit?resourcekey=0-5w8u2V-LS41tCHeoE8bDTQ", "type": 301 },
```

**Review by parlough — APPROVED** [review]
Perfect, thanks!

**Comment by bdero** [comment]
Hmm interesting, I didn't know stripping this resource key can cause problems. I'll fix this for my previous docs too.

**Comment by bdero** [comment]
> Hmm interesting, I didn't know stripping this resource key can cause problems. I'll fix this for my previous docs too.

Nevermind, the generated link for flutter.dev/go/impeller-geometry has no such key. 🤷‍♂️

**Pull request merged**